### ISSUE TITLE
Software with nothing to run on?

### ISSUE THREAD

**Issue by SoLongSidekick** [DjordjeMandic/Universal-IR-Blaster-TV-B-Gone#5]
That's my cheeky way of saying "you didn't include a damn link or any clue as to where/how one would buy the hardware you created to run this on". I was trying to find where I could purchase it and failed.

**Comment by DjordjeMandic** [comment]
That's true, this was small batch of prototypes from 2021. I'm currently busy with university but I was planning on making small production run to sell them. As soon as I get some free time (maybe from November onwards), I'm going to organize myself to make few boards. There will be more info about hardware, I'm already working on some documentation, improvements, ideas and expansion modules. I will inform you here
